### ISSUE TITLE
'module' object has no attribute 'SSL_ST_INIT' with OpenSSL 1.1.0

### ISSUE THREAD

**Issue by tdsmith** [pyca/pyopenssl#525]
`python -c "import OpenSSL.SSL"` fails on OS X 10.11 with Homebrew OpenSSL 1.1.0, cryptography 1.5, and PyOpenSSL 16.1.0.

Traceback looks like:
```
(tmp-fd20d2a1eb94ef4c) tim@rocketman:~$ python -c "import OpenSSL.SSL"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/tim/.virtualenvs/tmp-fd20d2a1eb94ef4c/lib/python2.7/site-packages/OpenSSL/__init__.py", line 8, in <module>
    from OpenSSL import rand, crypto, SSL
  File "/Users/tim/.virtualenvs/tmp-fd20d2a1eb94ef4c/lib/python2.7/site-packages/OpenSSL/SSL.py", line 112, in <module>
    SSL_ST_INIT = _lib.SSL_ST_INIT
AttributeError: 'module' object has no attribute 'SSL_ST_INIT'
```

Build looked like:
https://gist.github.com/tdsmith/6c411518f7bb162caebe848c6dee4bea

**Comment by reaperhulk** [comment]
This should be resolved with 16.2.0

**Comment by vhosakot** [comment]
I saw the `AttributeError: 'module' object has no attribute 'SSL_ST_INIT'` error too and `sudo pip install pyOpenSSL==16.2.0` resolved it for me!
